ospo-book/content/en/02-chapter.md

```diff
@@ -127,21 +142,22 @@ Assess this value for your own organization by taking steps such as:
 
-![opensourceinvolvementmodel](https://user-images.githubusercontent.com/43671777/232468143-cde69525-7adb-4399-96d3-fa63f056b942.png)
+`✅ Assessment`
 
-## Assessing value of open source contributions
+In this section, we will assess a 4-step process for communicating the value of contributing to open source and going beyond, which the OSPO can use when working with the different teams that engage with open source. (Source: [ospo-book mailing list discussion](https://lists.todogroup.org/g/WG-ospo-book-project/message/18)).
 
-`✅ Assessment`
+It is important to be clear that the following steps are not mandatory, but a set of recommended guidelines that organizations can adapt to assess the value of open source contributions.
 
-Despite an organization might be aware of the general problems, responsibility and benefits that contributing to open source provides, identifying specific key motivators to move people to take action (create activity) and prioritize open source is a tough task. In this section, we will assess a 4-step process for communicating the value of contributing to open source and going beyond, which the OSPO can use when working with the different teams that engage with open source. (Source: [ospo-book mailing list discussion](https://lists.todogroup.org/g/WG-ospo-book-project/message/18))
+### How do you assess open source activity engagement?
 
-## Step one: Assess open source activity engagement
+A good practice can be to make use of a maturity model to better map the different activities and present them to decision-makers. Thus, we recomend to get familiar with maturity models of open source adoption. These levels describe how open source is used in an increasingly effective fashion to drive value and address the organization's needs. One of the distinguishing factors for the different maturity levels is how open source contribution and creation are handled in an organization.
 
-Get familiar with maturity models of open source adoption. These levels describe how open source is used in an increasingly effective fashion to drive value and address organization's needs. One of the distinguishing factors for the different maturity levels is how open source contribution and creation are handled in an organization.
+There is a wide variety of open source maturity models –for governments, NGOs, Enterprises– with versions and subversions. In this book, we briefly share two of the most common ones used for open source adoption:
+
+#### Maturity Model 1 - Open source engagement adoption by [Dr. Ibrahim H](https://www.linuxfoundation.org/research/guide-to-enterprise-open-source)):
 
-For instance, a typical maturity model of corporate open source adoption looks like this (see this example by [Dr. Ibrahim H](https://www.linuxfoundation.org/research/guide-to-enterprise-open-source)):
 
 * Denial - No or unconscious use of open source
 * Consumption / Usage - Passive use of open source software
```

Comment:
I think we have lost some bullet points along the way?

Compare with https://ospobook.todogroup.org/02-chapter/ 

Denial - No or unconscious use of open source
Consumption / Usage - Passive use of open source software
Participation - Engagement with open source communities
Contribution - Pragmatic contributions to open source projects
Leadership - Strategic involvement with open source to drive business value

Reply:
you are right!

Reply:
```suggestion
* Consumption / Usage - Passive use of open source software
* Participation - Engagement with open source communities
* Contribution - Pragmatic contributions to open source projects
* Leadership - Strategic involvement with open source to drive business value
```